keyboards/cannonkeys/iron165/readme.md

```diff
@@ -0,0 +1,12 @@
+# Smith+Rune Iron165
+
+Iron165 Keyboard
+
+Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan)  
+Hardware Supported: STM32F072CBT6  
+
+Make example for this keyboard (after setting up your build environment):
+
+    make cannonkeys/iron165:default
```

Comment:
maybe a note about via support?

Reply:
I'll add those notes once they are officially in via !